spaceship/lib/spaceship/portal/certificate.rb

```diff
@@ -206,6 +206,23 @@ def create_certificate_signing_request
           return [csr, key]
         end
 
+        def create_apple_pay_certificate_signing_request
```

Comment:
Is this different from the other CSR logic we already have to have around somewhere in spaceship?

Reply:
@janpio  I don't think so. It's the same logic.

Reply:
Wouldn't it make sense to use that logic here then as well?

Reply:
@janpio actually, looks like the CSR logic for apple pay certificate is different than the one we currently have.

Current logic: https://github.com/RishabhTayal/fastlane/blob/2637eb2c673bb9fb7951bd9e3d8a5c1d4d58d21c/spaceship/lib/spaceship/portal/certificate.rb#L202-L202

Apple Pay logic: https://github.com/RishabhTayal/fastlane/blob/2637eb2c673bb9fb7951bd9e3d8a5c1d4d58d21c/spaceship/lib/spaceship/portal/certificate.rb#L214-L214

Reply:
So the next question is: Why does this have to be different? Is the certificate signing request different with newer requirements? Or doesn't this really matter and this is just the same thing, done differently?

Reply:
I am not exactly sure why it's different. Looks like Apple Pay signing certificate requires few more parameters to be sent for CSR. I don't want to change the existing `create_certificate_signing_request` method so I had to create a new one.

Reply:
Hm, that is still a bit unsatisfactory, isn't it?